src/CoinEntry.h

```diff
@@ -34,7 +34,10 @@ using SS58Prefix = uint32_t;
 /// Declare a dummy prefix to notify the entry to derive a delegated address.
 struct DelegatedPrefix {};
 
-using PrefixVariant = std::variant<Base58Prefix, Bech32Prefix, SS58Prefix, DelegatedPrefix, std::monostate>;
+/// Declare a dummy prefix to notify the entry to derive a firo exchange address.
+struct ExchangePrefix {};
```

Comment:
If it's necessary to support testnet exchange addresses, this `ExchangePrefix` structure could include `p2pkh` prefix (or also `p2sh` if applied).
What do you think?

Reply:
Thanks for the review. Actually, we do not have to support testnet now, if any requirement, we can extend this structure to support it. So I prefer keeping it empty to return mainnet address.